lib/progressive_hash_diff.rb

```diff
@@ -0,0 +1,30 @@
+class ProgressiveHashDiff
```

Comment:
Yeah, I can see that. I think if `ProgressiveHashDiff` were renamed to `HashDiffBuilder` it would make more sense to me.